website/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.0.0",
   "author": "HashiCorp",
   "devDependencies": {
-    "@hashicorp/platform-cli": "^1.2.0",
```

Comment:
I should check if this breaking upgrade affects anything else. @BRKalow if you have quick thoughts?

Reply:
Seems to me that the breaking change just upgrade the eslint version. So we should be fine.